org.adempiere.plugin.utils/src/org/adempiere/plugin/utils/PackInApplicationActivator.java

```diff
@@ -264,6 +264,7 @@ private File[] getFilesToProcess(String folders) {
 				continue;
 			}
 			
+			logger.warning("Processing " + filePath);
```

Comment:
this seems to contradict the name of this ticket.

Reply:
> this seems to contradict the name of this ticket.

Before this ticket the plugin writes one WARNING line for each file processed.

After this ticket the plugin writes one WARNING line for each folder, and one FINE line for each file processed.

I left the warning on folder to avoid making it mute, so people can still know that the AutomaticActivator is working.